deviceModeInit.js

```diff
@@ -171,11 +176,42 @@ var rudderTracking = (function () {
     };
     if (pages[path] === "Registration Viewed") {
       rudderanalytics.track(pages[path], properties);
+      rs$("#create_customer").submit(userRegistered);
+    } else if (pages[path] === "Login Viewed") {
+      rudderanalytics.track(pages[path], properties);
+      rs$("#customer_login").submit(userLoggedIn);
     } else {
       rudderanalytics.page(category, pageName, properties);
     }
   }
 
+
+  function userRegistered() {
+    const email = rs$('#create_customer [type="email"]').val();
+    const firstName = rs$(
+      '#create_customer [name="customer[first_name]"]'
+    ).val();
+    const lastName = rs$('#create_customer [name="customer[last_name]"]').val();
+    rudderanalytics.identify({
+      email,
+      firstName,
+      lastName,
+    });
+  }
+
+  function userLoggedIn() {
```

Comment:
Should we also track user log-out activity and clear the SDK cookies?

Reply:
I thought of that but it would need some research, as not all shops have the same logout button we can anchor to. We need to check if there is any common ground we can use